lib/app_builder.dart

```diff
@@ -33,12 +35,24 @@ class AppBuilder extends StatelessWidget {
               Provider<SearchService>(
                 create: (context) => SearchService(),
               ),
+              Provider<EventPageBloc>(
+                create: (context) => EventPageBloc(),
+                dispose: (context, bloc) => bloc.dispose(),
+              ),
+              Provider(
+                create: (context) => ViewAllBloc(
+                  eventService:
+                      Provider.of<EventService>(context, listen: false),
+                ),
+                dispose: (context, bloc) => bloc.dispose(),
+              ),
             ],
             child: builder(context, snapshot),
           );
         }
 
-        return builder(context, snapshot);
+        return builder(context,
+            snapshot); // TODO should populate with an error or fail screen
```

Comment:
Is this comment outdated?  If so can you explain what you mean by this.

Reply:
If for whatever reason the user credentials get lost between login and app build, this builder method gets called without an operating authService (I think). This comment is pointing out that this is technically an error state that could be handled with an error screen.